### ISSUE TITLE
Restrict a Container: Example section — incorrect code separation

### ISSUE THREAD

**Issue by nnlkcncff** [kubernetes/website#42348]
> https://kubernetes.io/docs/tutorials/security/apparmor/#example
> 
> ```bash
> kubectl create -f /dev/stdin <<EOF
> ```
> 
> ```yaml
> apiVersion: v1
> kind: Pod
> metadata:
>   name: hello-apparmor-2
>   annotations:
>     container.apparmor.security.beta.kubernetes.io/hello: localhost/k8s-apparmor-example-allow-write
> spec:
>   containers:
>   - name: hello
>     image: busybox:1.28
>     command: [ "sh", "-c", "echo 'Hello AppArmor!' && sleep 1h" ]
> EOF
> pod/hello-apparmor-2 created
> ```

<br>

The code block is split into two blocks, but it is a single command and so it should be written as one code block like this:

```bash
kubectl create -f /dev/stdin <<EOF
apiVersion: v1
kind: Pod
metadata:
  name: hello-apparmor-2
  annotations:
    container.apparmor.security.beta.kubernetes.io/hello: localhost/k8s-apparmor-example-allow-write
spec:
  containers:
  - name: hello
    image: busybox:1.28
    command: [ "sh", "-c", "echo 'Hello AppArmor!' && sleep 1h" ]
EOF
```

Output:
```bash
pod/hello-apparmor-2 created
```

**Comment by sftim** [comment]
I'd actually like to see us use a `code` shortcode - see https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files

**Comment by tengqm** [comment]
A simple fix would be just remove "pod/hello-apparmor-2 created" line.

**Comment by vaibhav2107** [comment]
I think the issue is resolved by the PR #45471 and hence we can close this issue
